playground-cli/index.js

```diff
@@ -0,0 +1,468 @@
+/* *******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the W3C Software Notice and
+ * Document License (2015-05-13) which is available at
+ * https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR W3C-20150513
+ ********************************************************************************/
+
+/**
+ * CLI interface for the Thing Description Playground
+ */
+const fs = require('fs')
+const path = require('path')
+const tdValidator = require('playground-core')
+const tdAssertions = require('playground-assertions')
+const assertManualToJson = require('playground-assertions').manualToJson
+const assertMergeResults = require('playground-assertions').mergeResults
+const assertCheckCoverage = require('playground-assertions').checkCoverage
+const assertResultsToCsv = require('playground-assertions').resultsToCsv
+const argParser = require('argly')
+    .createParser({
+        '--help': {
+            type: 'string',
+            description: 'You can call the playground validation with no input (example folder will be taken), \n'+
+                        'a Thing Description (.json file), a folder with multiple Thing Descriptions, \n' +
+                        'or a Folder with "valid", "invalid" and "warning" subfolder, where all included TDs \n' +
+                        'will be checked whether they produce the expected validation result.'
+        },
+        '--input -i *': {
```

Comment:
the description should specify that this only does core validation of TDs

Reply:
https://github.com/thingweb/thingweb-playground/pull/125/commits/263803b0119d4157e4f946ed70f9e9efc684d7bc